src/amo/components/SearchFilters/index.js

```diff
@@ -245,6 +289,51 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
             </div>
           )}
 
+          {filters.addonType === ADDON_TYPE_STATIC_THEME && (
+            <div>
+              <label
+                className="SearchFilters-label SearchFilters-Color-label"
+                htmlFor="SearchFilters-CustomColor"
+              >
+                {i18n.gettext('Theme Color')}
+              </label>
+              <ul className="SearchFilters-ThemeColors">
+                {colorFilters.map((colorFilter) => (
+                  // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
+                  <li
+                    key={colorFilter.id}
+                    className={makeClassName(
+                      'SearchFilter-ThemeColors-ColorItem',
+                      `SearchFilter-ThemeColors-ColorItem--${colorFilter.id}`,
+                      {
+                        'SearchFilter-ThemeColors-ColorItem--active':
+                          filters.color === colorFilter.color,
+                      },
+                    )}
+                    data-color={colorFilter.color}
```

Comment:
You can pass the color directly as an argument to onColorClick. Why use a data attribute? Seems like more code and more dom manipulation for no apparent benefit.

```js
onClick={() => this.onColorClick(colorFilter.color)}
```

This fragment will not re-render very often so there is little to no marginal cost for the anonymous function.